src/main/java/g1501_1600/s1545_find_kth_bit_in_nth_binary_string/Solution.java

```diff
@@ -3,7 +3,7 @@
 // #Medium #String #Recursion #2022_04_11_Time_0_ms_(100.00%)_Space_41.2_MB_(90.34%)
 
 public class Solution {
-    public char findKthBit(int n, int k) {
+    public char findKthBit(int k) {
```

Comment:
Please add SuppressWarnings annotation for non used argument.

Reply:
The code is java:S1172

Reply:
thanks